dotnet/src/InternalUtilities/src/Schema/KernelJsonSchemaBuilder.cs

```diff
@@ -22,16 +22,20 @@ namespace Microsoft.SemanticKernel;
 internal static class KernelJsonSchemaBuilder
```

Comment:
I didn't remove this class to minimize churn, but at this point it seems reasonable to bake in this factory into `KernelJsonSchema` (and perhaps make it public).